WordPress/WordPressShareExtension/String+Extensions.swift

```diff
@@ -88,6 +88,15 @@ extension String {
         return self + returnURLString
     }
 
+    /// Returns a Boolean value indicating if this String begins with provided prefix.
+    /// - Parameter prefix: The provided prefix to verify.
+    /// - Parameter options: The provided compare options.
```

Comment:
💄  Not sure if this is some documentation convention/habit but I found it quite original that you use "provided" in the parameters documentation… the parameters are always something that the caller is provided, so felt a bit unusual to have that wording here 😄 

What do you think of this instead?
```suggestion
    /// Returns a Boolean value indicating if this String begins with the provided prefix.
    /// - Parameter prefix: The prefix to check for.
    /// - Parameter options: The string comparison options to use when checking for the prefix.
```

Reply:
I checked other documentation around the project, and `provided` was used a few times. I agree with you that it can be improved. I'm going to apply your suggestion. Thank you!